core/src/main/java/com/google/cloud/sql/core/CoreSocketFactory.java

```diff
@@ -291,24 +263,11 @@ private static String getUserAgents() {
   }
 
   /** Returns the current User-Agent header set for the underlying SQLAdmin API client. */
-  public static String getApplicationName() {
+  private static String getApplicationName() {
     if (coreSocketFactory != null) {
       return coreSocketFactory.adminApiService.getApplicationName();
     }
-    return System.getProperty(USER_TOKEN_PROPERTY_NAME, "");
-  }
-
-  /**
-   * Sets the User-Agent header for requests made using the underlying SQLAdmin API client.
-   *
-   * @throws IllegalStateException if the SQLAdmin client has already been initialized
-   */
-  public static void setApplicationName(String applicationName) {
```

Comment:
Didn't we see usage of this in a Spring artifact?

Reply:
You are correct. I deleted this in haste. I've updated the design doc accordingly.

Reply:
I think we should work on removing this from spring cloud too. In theory, we can support appending the user agent, but otherwise not allow wholesale replacement of it.

Reply:
I agree. When I update the user agent behavior, this value will be appended to the UA.